tuna/utils/db_utility.py

```diff
@@ -198,6 +226,18 @@ def gen_select_objs(session, attribs, tablename, cond_str):
   return entries
 
 
+#def get_db_obj_by_id(row_id, db_class):
```

Comment:
Do we still need this function in the future?

Reply:
i dont think so, ill look to remove